detection-rules/recon_recipients_bccd_undisclosed.yml

```
@@ -0,0 +1,43 @@
+name: "Reconaissance: All recipients bcc'd or undisclosed"
+description: |
+  Recon messages are used to validate whether a recipient address is valid or not, potentially preceding an attack.
```

```suggestion
  Recon messages, a form of deliverability testing, are used to validate whether a recipient address is valid or not, potentially preceding an attack.
```

Just trying to work in the buzzwords, could adopt here or in the tags. I'll suggest both